roles/sap_ha_cluster/tasks/ascertain_ha_cluster_in_inventory.yml

```diff
@@ -0,0 +1,59 @@
+---
+# If there are 'ha_cluster" role parameters are already defined in the
+# inventory, we will use these custom specifications.
+
+## Take the following template to add additional parameters to be inherited:
+#- name: "{{ __tname }} Register ha_cluster_"
+#  ansible.builtin.set_fact:
+#    __sap_ha_cluster_: "{{ ha_cluster_ }}"
+#  when: ha_cluster_ is defined
+
+- name: "{{ __tname }} Register ha_cluster_cluster_properties"
+  ansible.builtin.set_fact:
+    __sap_ha_cluster_cluster_properties: "{{ ha_cluster_cluster_properties }}"
```

Comment:
Please include the "_fact" string in all variables which are set by set_fact, as per https://sap-linuxlab.github.io/contribute/contribute_04_dev_guidelines/.

Reply:
internal variables only, so this can be done later.